src/platform/BUILD.gn

```diff
@@ -289,6 +289,11 @@ if (chip_device_platform != "none" && chip_device_platform != "external") {
         "CHIP_DEVICE_LAYER_TARGET_OPEN_IOT_SDK=1",
```

Comment:
mode of this file should not be changed. It changed to 755 from 644. Please do not make files executable.